app/templates/components/repo-actions.hbs

```diff
@@ -66,4 +66,8 @@
       </div>
     {{/if}}
   {{/if}}
+{{else}}
+  {{#if this.islist}}
+    <LoadingIndicator @height={{true}} />
```

Comment:
Doesn't it look good even without `height` and styles tweaking? Here's how it looks to me (I tested only in chrome though):

![2019-11-12 at 17 10](https://user-images.githubusercontent.com/4066489/68678744-935bd800-056f-11ea-8292-d102e1c05650.gif)

Did you maybe notice something that I missed? 😅

Reply:
You are absolutely right. Thank you @artursmirnov